instance/models/instance.py

```diff
@@ -89,6 +92,18 @@ def save(self, *args, **kwargs):
             'instance_id': self.pk,
         })
 
+    @classmethod
+    def can_manage(cls, user):
+        """
+        Returns true if the user is allowed to manage all instances.
+
+        User must:
+        * be a staff member, or
+        * have the "manage_all" permission
+        """
+        permission = '{}.{}'.format(cls._meta.app_label, "manage_all")
+        return user.is_staff or user.has_perm(permission)
```

Comment:
@bradenmacdonald I took this requirement from [OC-1550](https://tasks.opencraft.com/browse/OC-1550) to mean it needed to include is_staff users as well as people with the permission:

> Make sure staff users (is_staff = True) get access to everything as well.